rj_gameplay/rj_gameplay/action/intercept.py

```diff
@@ -0,0 +1,38 @@
+"""This module contains the interface and action for intercept."""
```

Comment:
Broken intercept, not sure if it's motion planning side or here. Feel free to skip for now since it's not used.